app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>PrivateEvents</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+  <body>
+    <div class="window">
+      <div class="title-bar">
+        <div class="title-bar-text">An event manager from the future</div>
+        <div class="title-bar-controls">
+          <% if user_logged_in %>
+            <%= link_to destroy_session_path, method: :delete do %>
+              <button aria-label="Close" id='logout'></button>
+            <% end %>
+          <% end %>
```

Comment:
- [ ] put the if statement in a helper method that renders the link instead of having this complexity directly in the view